.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -0,0 +1,114 @@
+      options:
+        - label: I'm using the latest version of `pact-python`.
+          required: true
+        - label: I have read the console error message carefully (if applicable).
+
+  - type: textarea
+    attributes:
+      label: Description
+      description: A clear and concise description of what the bug is.
+    validations:
+      required: true
+
+  - type: input
+    attributes:
+      label: Reproducible demo
+      description: |
+        Paste the link to an example repo if possible, and exact instructions to reproduce the issue.
+
+        > **What happens if you skip this step?** Someone will read your bug report, and maybe will be able to help you, but it's unlikely that it will get much attention from the team. Eventually, the issue will likely get closed in favor of issues that have reproducible demos.
+
+        Please remember that:
+
+        - Issues without reproducible demos have a very low priority.
+        - The person fixing the bug would have to do that anyway. Please be respectful of their time.
+        - You might figure out the issues yourself as you work on extracting it.
+
+        Thanks for helping us help you!
+
+  - type: textarea
+    attributes:
+      label: Steps to reproduce
+      description: Write down the steps to reproduce the bug. You should start with a fresh installation, or your git repository linked above.
+      placeholder: |
+        1. Step 1...
+        2. Step 2...
+        3. Step 3...
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: Expected behavior
+      description: How did you expect your project to behave? It's fine if you're not sure your understanding is correct. Write down what you thought would happen.
+      placeholder: Write what you thought would happen.
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: Actual behavior
+      description: |
+        Did something go wrong? Is something broken, or not behaving as you expected? Describe this section in detail. Don't only say "it doesn't work"! Please submit exhaustive and complete log messages, not just the final error message.
+
+        > Please read error messages carefully: it often tells you exactly what you are doing wrong.
+
+        If the logs are too long, you can paste them in a [gist](https://gist.github.com/) and link it here.
+      placeholder: |
+        Write what happened. Add full console log messages.
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: Your environment
+      description: Include as many relevant details about the environment you experienced the bug in.
+      value: |
+        - Public source code:
+        - Pact Python version used: `pip list | grep pact`
+        - Operating system and version (e.g. Ubuntu 20.04.2 LTS, macOS Ventura):
+
+  - type: checkboxes
+    attributes:
+      label: Self-service
```

Comment:
i loved this bit from docusaurus!

Reply:
Yeah me too! I think it's a fantastic way to invite contribution, especially if they have gotten through all the steps to submit a bug report! They might have a fair idea of how to fix it already.